hapi/src/api/user.api.js

```diff
@@ -44,7 +44,6 @@ const SET_NAME = `
 
 const getOne = async (where = {}) => {
   const { user } = await hasuraUtils.request(GET_ONE, { where })
-
   if (user && user.length > 0) {
```

Comment:
this **if statement** could be a single line

`if (user && user.length > 0) return user[0]`